File: android/app/src/main/java/com/example/morpho_sample/info/subtype/SecurityOption.java
// The present software is not subject to the US Export Administration Regulations (no exportation license required), May 2012
package com.example.morpho_sample.info.subtype;

public class SecurityOption
{
	public boolean	activated	= false;
	public String	title		= "";

	public SecurityOption(boolean activated, String title)
	{
		this.activated = activated;
		this.title = title;
	}

	public String toString(String no, String yes)
	{
		String act = no;
		if (activated)
			act = yes;
		return act + "\t" + title;
	}
}

File: android/app/src/main/kotlin/my/com/tbs/jpj/qto/app/info/subtype/SecurityOption.kt
// The present software is not subject to the US Export Administration Regulations (no exportation license required), May 2012
package my.com.tbs.jpj.qto.app.info.subtype

class SecurityOption(activated: Boolean, title: String) {
    var activated = false
    var title = ""
    fun toString(no: String, yes: String): String {
        var act = no
        if (activated) act = yes
        return act + "\t" + title
    }

    init {
        this.activated = activated
        this.title = title
    }
}